### PR TITLE
BIOWDL-390: Setup PacBio subreads processing pipeline.

### PR DESCRIPTION
**Pull request by JasperBoom**

### CODE REVIEW

**Comment by JasperBoom** [comment]
I have kept in the "samplesheet" input json for now. It allows me to make sure that the correct barcode fasta file is linked to its associated subreads input bam file.

File: PacBio-subreads-processing.wdl [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import "structs.wdl" as structs
+import "tasks/biowdl.wdl" as biowdl
+import "tasks/common.wdl" as common
+import "tasks/ccs.wdl" as ccs
+import "tasks/lima.wdl" as lima
+import "tasks/isoseq3.wdl" as isoseq3
```
Comment (DavyCats) [inline PacBio-subreads-processing.wdl]:
```suggestion
import "tasks/ccs.wdl" as ccs
import "tasks/common.wdl" as common
import "tasks/isoseq3.wdl" as isoseq3
import "tasks/lima.wdl" as lima
```

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+```json
+{"subreads": [{"subreads_id": "id",
+"subreads_file": "path/to/subreads.bam",
+"subreads_md5": "94127ced6d8428301376ee4ac18df58a",
+"barcodes_file": "path/to/barcodes.fasta"},
+{"subreads_id": "id",
+"subreads_file": "path/to/subreads.bam",
+"subreads_md5": "94127ced6d8428301376ee4ac18df58a",
+"barcodes_file": "path/to/barcodes.fasta"}]}
```
Comment (DavyCats) [inline docs/index.md]:
```suggestion
{
  "subreads": [
    {
      "subreads_id": "id",
      "subreads_file": "path/to/subreads.bam",
      "subreads_md5": "94127ced6d8428301376ee4ac18df58a",
      "barcodes_file": "path/to/barcodes.fasta"
    }, {
      "subreads_id": "id2",
      "subreads_file": "path/to/subreads2.bam",
      "subreads_md5": "94127ced6d8428301376ee4ac18df58b",
      "barcodes_file": "path/to/barcodes2.fasta"
    }
  ]
}
```

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+The workflow will output polished CCS reads split into their respective sample.
+Along with these BAM files, the workflow will also output all intermediate
+generated files.
+
+## Contact
+<p>
+  <!-- Obscure e-mail address for spammers -->
+For any questions about running this pipeline and feature request (such as
+adding additional tools and options), please use the
+<a href='https://github.com/biowdl/sequence-classification/issues'>github issue tracker</a>
```
Comment (DavyCats) [inline docs/index.md]: Incorrect link

**Review by DavyCats — CHANGES_REQUESTED** [review]

**Review by DavyCats — APPROVED** [review]

**Pull request merged**